tests/test_bcc_remote.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+from bcc.remote import shell
+
+s = shell.ShellRemote()
+print s.send_command('testxx')
```

Comment:
is this test really expected to print something? And which commit has all the smoke tests your wrote??

Reply:
I intended the tests to come later as a separate commit.

But now that you mention it, it probably is better to include the tests in this commit instead. I'll apply this change shortly.